spec/dummy/Gemfile.lock

```diff
@@ -11,67 +11,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (7.0.0)
-      actionpack (= 7.0.0)
-      activesupport (= 7.0.0)
+    actioncable (7.0.4.2)
```

Comment:
gem version items should be in separate PR!

Reply:
I reverted the commit. I am not sure why I pushed that commit. I shouldn't.